examples/app_engine/main.tf

```
@@ -14,23 +14,34 @@
  * limitations under the License.
  */
 
-locals {
-  credentials_file_path = "${path.module}/sa-key.json"
+provider "google" {
+  version = "~> 2.18.1"
 }
 
-/******************************************
-  Provider configuration
- *****************************************/
-provider "gsuite" {
-  credentials = file(local.credentials_file_path)
-  version     = "~> 0.1.12"
+provider "google-beta" {
+  version = "~> 2.18.1"
+}
+
+resource "random_string" "suffix" {
+  length  = 4
+  special = false
+  upper   = false
+}
+
+module "app-engine-project" {
+  source            = "../../"
+  name              = "appeng-${random_string.suffix.result}"
+  random_project_id = "true"
```

Booleans no longer require quotation marks.

```suggestion
  random_project_id = true
```

---

Done